tgui/packages/tgui/interfaces/Keybinds/formatKeyboardEvent.ts

```diff
@@ -44,8 +43,8 @@ export const formatKeyboardEvent = (event: React.KeyboardEvent): string => {
 
   if (isStandardKey(event)) {
     const key = event.key.toUpperCase();
-    text += KEY_CODE_TO_BYOND[key] || key;
+    text += KEY_CODE_TO_BYOND[key] || key === ' ' ? 'Space' : key;
   }
 
-  return text;
+  return text.replace(/\+$/, '');
```

Comment:
Could we get a comment here explaining why we're trimming trailing `+` characters? Or, like, why it might have them, etc.

Reply:
Code improved to only add `+`s when needed, so they're present for bindings such as "SHIFT+S" but absent for only "SHIFT"